phpBB/includes/functions_user.php

```diff
@@ -1709,6 +1709,14 @@ function validate_username($username, $allowed_username = false)
 		return false;
 	}
 
+	// The very first check is for
+	// out-of-bounds characters that are currently
+	// not supported by utf8_bin in MySQL
+	if (preg_match('/[\x{10000}-\x{10FFFF}]/u', $username))
+	{
+		return 'INVALID_EMOJIS_USERNAME';
```

Comment:
It should just return `INVALID_EMOJIS` sine `_USERNAME` is appended to the result of this function

Reply:
Done, thx.